docs/reference/migration/migrate_7_0/mappings.asciidoc

```diff
@@ -52,3 +52,19 @@ as a better alternative.
 
 An error will now be thrown when unknown configuration options are provided
 to similarities. Such unknown parameters were ignored before.
+
+[float]
+==== deprecated `geo_shape` Prefix Tree indexing
+
+`geo_shape` types now default to using a vector indexing approach based on Lucene's new
+`LatLonShape` field type. This indexes shapes as a triangular mesh instead of decomposing
+them into individual grid cells. To index using legacy prefix trees `recursive` or `term`
+strategy must be explicitly defined. Note that these strategies are now deprecated and will
+be removed in a future version.
+
+[float]
+==== deprecated `geo_shape` parameters
+
+The following type parameters are deprecated for the `geo_shape` field type: `tree`,
+`precision`, `tree_levels`, `distance_error_pct`, `points_only`, and `strategy`. They
+will be removed in a future version.
```

Comment:
these changes should be in `docs/reference/migration/migrate_6_6.asciidoc` instead? (only exists on the 6.x branch)

Reply:
Yes I agree.... I can do that in the backport? Or should I open a whole new PR to handle that?